### PR TITLE
Verified and corrected proptypes and naming conventions in all 3 comp…

### PR DESCRIPTION
**Pull request by ibxibx**
double-checked the correct proptypes and naming conventions

### CODE REVIEW

File: src/components/movie-view/movie-view.jsx [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import PropTypes from "prop-types";
-
```
Comment (stratospherique) [inline src/components/movie-view/movie-view.jsx]: This line shouldn't be removed. It's causing the app to break.

Check the proptype definition section in the bottom. There is usage of `PropTypes` class. Therefore, it should be imported at the top of the script.

File: src/components/movie-view/movie-view.jsx [hunk 2]
```diff
@@ -35,23 +33,22 @@ MovieView.propTypes = {
   movie: PropTypes.shape({
     _id: PropTypes.string.isRequired,
     Title: PropTypes.string.isRequired,
+    ImagePath: PropTypes.string.isRequired,
+    Director: PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      bio: PropTypes.string.isRequired,
+      birthYear: PropTypes.number.isRequired,
```
Comment (stratospherique) [inline src/components/movie-view/movie-view.jsx]: If you check the browser's devtool > console tab, you'll notice a runtime warning displayed about birthYear being expected but received as undefined. It means the the birthYear of the director in movie object is either unexistant or nil. Please make sure to know your data and properly define the proptypes.

**Review by stratospherique — CHANGES_REQUESTED** [review]

File: src/components/movie-view/movie-view.jsx (hunk 1 above)
Comment (ibxibx) [reply src/components/movie-view/movie-view.jsx]: Yes, I've put it back.

File: src/components/movie-view/movie-view.jsx (hunk 2 above)
Comment (ibxibx) [reply src/components/movie-view/movie-view.jsx]: Solved the issue.

**Pull request merged**